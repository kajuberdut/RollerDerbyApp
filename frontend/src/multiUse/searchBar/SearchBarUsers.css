.SearchBarUsers {
    display: flex;   
}

.SearchBarUsers-City {
    padding-left: 5px;
    padding-right: 5px;
}

.SearchBarUsers-State {
    padding-left: 5px;
    padding-right: 5px;
}

.SearchBarUsers-Username {
    padding-left: 5px;
    padding-right: 5px;
}


.SearchBarInput {
    border: 1px solid rgb(100, 97, 97);
    height: 30px;
    border-radius: 7%;
}

.SearchBarInput:focus {
    border-color: black !important;
    border-width: 2px !important;
    box-shadow: none !important;
    border-radius: 7% !important;
    padding: none !important;
    margin: none !important;
  }
  
#state {
    width: 87px !important;
    height: 30px !important;
    padding-top: 2px !important;
    padding-bottom: 2px !important;
}

#city {
    width: 120px;
}

#username {
    width: 120px;
}
