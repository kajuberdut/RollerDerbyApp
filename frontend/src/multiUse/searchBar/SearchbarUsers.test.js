import { render, screen } from '@testing-library/react';
import SearchBarUsers from './SearchBarUsers';

// npm test SearchBarUsers.test.js
// must be in frontend directory 

const formData =  jest.fn().mockResolvedValue(
  {
    username: "User1"
  }
)

test('renders SearchbarUsers', () => {
  render(<SearchBarUsers formData={formData} />);
});


test('matches snapshot', function() {
  const { asFragment } = render(<SearchBarUsers formData={formData} />);
  expect(asFragment()).toMatchSnapshot(); 
})

test('renders SearchbarUsers details', () => {
  render(<SearchBarUsers formData={formData}  />);

  expect(screen.getByPlaceholderText('Derby Name')).toBeInTheDocument();
});