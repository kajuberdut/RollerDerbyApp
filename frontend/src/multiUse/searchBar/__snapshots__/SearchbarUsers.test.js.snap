// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="SearchBarUsers"
  >
    <div
      class="SearchBarUsers-Username"
    >
      <input
        class="SearchBarInput form-control"
        id="searchbar-username"
        name="username"
        placeholder="Derby Name"
        type="text"
        value=""
      />
    </div>
  </div>
</DocumentFragment>
`;
